version: "3.8"

services:
  dynamodb:
    image: amazon/dynamodb-local
    restart: on-failure:3
    volumes:
      - dynamodb:/home/dynamodblocal/data
    ports:
      - 8000:8000

volumes:
  dynamodb:
    external: false
